src/platform-includes/performance/opentelemetry-setup/python.mdx

```diff
@@ -0,0 +1,26 @@
+Initialize Sentry for tracing and set the `instrumenter` to `otel`:
+
+```python
+import sentry_sdk
+
+sentry_sdk.init(
```

Comment:
Is anything else required for linking errors to OTEL transactions? e.g. adding an `EventProcessor`?

Reply:
The other imports are below (this is in another mdx file) see the full page here in the preview: https://sentry-docs-git-antonpirker-opentelementry-support-python-sdk.sentry.dev/platforms/python/performance/instrumentation/opentelemetry/

(Previews are linked in the comment from the vercel bot above)